### ISSUE TITLE
Lots of errors and no functionality

### ISSUE THREAD

**Issue by heggink** [SevenW/Plugwise-2-py#38]
Hi SevenW,

For the second time now in a couple of months, everything gets stuck and I get lots of errors in pw-logger.log:

2017-05-17 23:52:19,470 - pw-logger - ERROR - unexpected response [1]:'expected response code 0013, received code 0000'
2017-05-17 23:52:19,472 - pw-logger - INFO - OFFLINE Circle 'Dryer'.
2017-05-17 23:52:19,473 - pw-logger - ERROR - Error in ten_seconds(): Timeout while waiting for response from circle 'Dryer'
2017-05-17 23:52:24,274 - pw-logger - ERROR - unexpected response [1]:'expected response code 0013, received code 0000'
2017-05-17 23:52:24,276 - pw-logger - INFO - OFFLINE Circle 'Washer'.
2017-05-17 23:52:24,277 - pw-logger - ERROR - Error in ten_seconds(): Timeout while waiting for response from circle 'Washer'
2017-05-17 23:52:24,600 - pw-logger - INFO - 000D6F00019E1B1C: save log 
2017-05-17 23:52:24,841 - pw-logger - INFO - circle buffers: 000D6F00019E1B1C 3264 2017-05-17 22:50 36 6
2017-05-17 23:52:24,844 - pw-logger - INFO - circle buffers: 000D6F00019E1B1C Raspi read from 3264 to 3264
2017-05-17 23:52:30,429 - pw-logger - INFO - 000D6F0001A44750: save log 
2017-05-17 23:52:40,531 - pw-logger - INFO - 000D6F00005528D5: save log 
2017-05-17 23:52:55,380 - pw-logger - ERROR - unexpected response [1]:'expected response code 0013, received code 0000'
2017-05-17 23:52:55,382 - pw-logger - INFO - OFFLINE Circle 'Freezer'.
2017-05-17 23:52:55,384 - pw-logger - ERROR - Error in ten_seconds(): Timeout while waiting for response from circle 'Freezer'
2017-05-17 23:52:55,451 - pw-logger - INFO - 000D6F00028F6122: save log 
2017-05-17 23:53:00,288 - pw-logger - ERROR - unexpected response [1]:'expected response code 0013, received code 0000'
2017-05-17 23:53:00,290 - pw-logger - ERROR - Out of sequence message. Expected seqnr 0349, received seqnr 0344
2017-05-17 23:53:00,291 - pw-logger - ERROR - TEST: 0000
2017-05-17 23:53:00,292 - pw-logger - ERROR - unexpected response [4]:'expected response code 0013, received code 0000'
2017-05-17 23:53:00,425 - pw-logger - ERROR - unexpected response [1]:'expected response code 0013, received code 000E'
2017-05-17 23:53:00,427 - pw-logger - ERROR - TEST: 000E
2017-05-17 23:53:00,428 - pw-logger - ERROR - unexpected response [4]:'expected response code 0013, received code 000E'
2017-05-17 23:53:00,442 - pw-logger - ERROR - unexpected response [1]:'expected response code 0013, received code 000E'
2017-05-17 23:53:00,443 - pw-logger - ERROR - TEST: 000E
2017-05-17 23:53:00,444 - pw-logger - ERROR - unexpected response [4]:'expected response code 0013, received code 000E'
2017-05-17 23:53:00,566 - pw-logger - INFO - 000D6F0001A44645: save log 
2017-05-17 23:53:10,395 - pw-logger - INFO - expect_response: out of sequence ping response
2017-05-17 23:53:10,403 - pw-logger - INFO - expect_response: out of sequence ping response
2017-05-17 23:53:10,657 - pw-logger - INFO - 000D6F00019E07EF: save log 
2017-05-17 23:53:10,735 - pw-logger - INFO - expect_response: out of sequence ping response
2017-05-17 23:53:10,976 - pw-logger - INFO - Circle 000D6F00019E07EF in state pong. Bring to online.
2017-05-17 23:53:15,502 - pw-logger - ERROR - unexpected response [1]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,504 - pw-logger - ERROR - Out of sequence message. Expected seqnr 035F, received seqnr 0358
2017-05-17 23:53:15,505 - pw-logger - ERROR - TEST: 0000
2017-05-17 23:53:15,507 - pw-logger - ERROR - unexpected response [4]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,511 - pw-logger - ERROR - unexpected response [1]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,513 - pw-logger - ERROR - Out of sequence message. Expected seqnr 035F, received seqnr 0359
2017-05-17 23:53:15,514 - pw-logger - ERROR - TEST: 0000
2017-05-17 23:53:15,515 - pw-logger - ERROR - unexpected response [4]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,632 - pw-logger - ERROR - unexpected response [1]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,633 - pw-logger - ERROR - Out of sequence message. Expected seqnr 035F, received seqnr 035B
2017-05-17 23:53:15,635 - pw-logger - ERROR - TEST: 0000
2017-05-17 23:53:15,636 - pw-logger - ERROR - unexpected response [4]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,648 - pw-logger - ERROR - unexpected response [1]:'expected response code 003F, received code 0000'
2017-05-17 23:53:15,649 - pw-logger - ERROR - Out of sequence message. Expected seqnr 035F, received seqnr 035C
2017-05-17 23:53:15,651 - pw-logger - ERROR - TEST: 0000
2017-05-17 23:53:15,652 - pw-logger - ERROR - unexpected response [4]:'expected response code 003F, received code 0000'

Basically nothing works, no circles are available. No clue what triggers it...

Anything I can do to debug?

H

**Comment by heggink** [comment]
NVM, found the problem: 2 ftdi devices all of a sudden swapped names (/dev/ttyUSB0 and USB1) where another device now wanted access to the stick.
